ops/dev/persistent/main.tf

```diff
@@ -16,7 +16,7 @@ module "monitoring" {
   rg_location   = data.azurerm_resource_group.dev.location
   rg_name       = data.azurerm_resource_group.dev.name
 
-  app_url = "${local.env}.simeplreport.gov"
+  app_url = "${local.env}.simplereport.gov"
```

Comment:
😱 

This appears to be replicated in 8 other files. We should fix that. Also how has that not caused a problem? Or has it, and we didn't realize?

I feel like this should actually appear in fewer places anyway, since it's a variable interpolation. Also it should be spelled right.

Reply:
Hah, I wondered the same thing. So, a few things here:

- It looks like we're setting variables based on which directory the file is in rather than basing it off the current env (Terraform workspace, in this case), so there's a lot of copy/paste going on. It might be possible to have `ops/services/monitring/main.tf` generate this for us if the URL changes are uniform between environments (e.g. it's a simple interpolation of the current env or something like that).
- It doesn't currently matter anyway because 1) we're not actually passing that variable into the monitoring module anyway and 2) the monitoring module is accepting that variable being passed in but it is _also_ not doing anything with it. So, we've never noticed the typo because it's not being referenced.

Reply:
Just pushed a commit to do a find/replace for simepl -> simple (and a few instances of simeple)